testhelpers/docker_registry.go

```diff
@@ -68,17 +68,24 @@ func NewDockerRegistry(ops ...RegistryOption) *DockerRegistry {
 }
 
 // BasicAuth wraps a handler, allowing requests with matching username and password headers, otherwise rejecting with a 401
-func BasicAuth(handler http.Handler, username, password, realm string) http.Handler {
+// optPermissions is used to skip the authentication check if the image is readable.
```

Comment:
```suggestion
// optPermissions is used to skip the authentication check if the image has been configured in the test setup to be always readable.
```

Reply:
Fixed in the following [commit](https://github.com/buildpacks/imgutil/pull/126/commits/1a4373ac85d225b6bd3771c3ccadae2a239cf6db)